src/Pages/GlobalConfigurations/Authorization/APITokens/RegenerateModal.tsx

```diff
@@ -148,6 +148,7 @@ const RegeneratedModal = ({
                     onSave={handleRegenrateToken}
                     buttonText="Regenerate Token"
                     regenerateButton
+                    disabled={false}
```

Comment:
is this required?

Reply:
Yes, added it as a required prop on the component since it is being used at multiple places